net/nfs-kernel-server/patches/host/10-macos-build-hack.patch

```diff
@@ -0,0 +1,55 @@
+diff --git a/configure b/configure
```

Comment:
You should send this to upstream. You even mention that it is a hack in the file naming, but we should not have hacks forever, otherwise we will take care of it forever.